jobs/config.json

```diff
@@ -10461,6 +10461,30 @@
       "sig-network"
     ]
   },
+  "pull-kubernetes-e2e-containerd-gce": {
+    "args": [
+      "--build",
```

Comment:
you don't need this (already have --build=bazel)

Reply:
Done.